src/app/core/services/chat.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
```

Comment:
Good implementation of the chat method to be used to create a chat service, Joseph!